backend/typescript/middlewares/validators/camperValidators.ts

```diff
@@ -211,34 +211,6 @@ export const updateCamperDtoValidator = async (
   if (req.body.allergies && !validatePrimitive(req.body.allergies, "string")) {
     return res.status(400).send(getApiValidationError("allergies", "string"));
   }
-  if (!Array.isArray(req.body.earlyDropoff)) {
```

Comment:
I'm a bit confused can you explain why we're deleting all of this? Is it just not necessary?

Reply:
these fields can't be edited once the camper is registered. I took them out of the `UpdateCamperDTO` object too. Currently, the validator was looking for those fields but since they can't be edited, it doesn't make sense to include them.

Reply:
Ah now that makes sense to me.